OpenEdXMobile/res/values/strings.xml

```diff
@@ -467,11 +467,10 @@
     <!-- Prompt leading user to sign in screen -->
     <string name="already_have_an_account">Already have an account? Sign in</string>
     <!-- Prompt leading user to create account screen -->
-    <string name="register_and_learn">Register and start learning</string>
-    <!-- In Register screen -->
-    <string name="register_with">Register with</string>
-    <!-- In register screen -->
-    <string name="or_register_with_email">Or register with email</string>
+    <string name="sign_up_and_learn">Sign up and start learning</string>
+    <!-- In Sign Up screen -->
+    <string name="sign_up_with">Sign up with</string>
```

Comment:
This within the scope of this PR and needs to be reverted to what was originally discussed. 
`Sign up` -> `Register`